server/monitoring/monitoring.go

```diff
@@ -48,6 +51,7 @@ func Monit() {
 		} else {
 			if found > 0 {
 				logger.Info("monitoring", fmt.Sprintf("Deleted %d entries and files in %s", found, time.Since(now)))
+				statistics.S.UpdateCurrentSize(sizeRemoved)
```

Comment:
Are you sure you're not updating to the wrong size ? Might be more something like `statistics.S.UpdateCurrentSize(statistics.S.CurrentSize-sizeRemoved)` ?